### PR TITLE
Upgrade terraform-libvirt-provider to 0.6.12

### LINKED ISSUE #15: Upgrade to v0.6.12 of the Libvirt Terraform Provider

### PR DESCRIPTION
**Pull request by dustinspecker**
Closes #15

I hope it's cool I took a stab at this. I was running into a bug with bridge mode and qemu-agent not working properly but noticed https://github.com/dmacvicar/terraform-provider-libvirt/pull/873 fixes the problem. After updating to 0.6.12, I've confirmed it fixed the problem for me using pulumi-libvirt.

I haven't done any Pulumi/Terraform dev before, so please let me know if I can help with anything else or if I'm way off base.

The `CONTRIBUTING.md` and `Makefile` were super helpful! I wasn't sure how to get Pulumi to use a forked plugin binary, so I just manually copied the locally built provider binary over the existing `resource-libvirt-v0.2.0` binary in `~/.pulumi` and that seemed to work to prove the bridge/qemu-agent bug was fixed.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ CHANGELOG
 
 ## HEAD (Unreleased)
 _(none)_
```
Comment (jkodroff) [inline CHANGELOG.md:5]: This line has to be deleted.

**Review by jkodroff — CHANGES_REQUESTED** [review]

**Comment by jkodroff** [comment]
@dustinspecker Thanks for the contribution!  Looks good, other than the CHANGELOG issue noted above.  If you push changes, we should be good to go, pending the checks passing.

File: CHANGELOG.md (hunk 1 above)
Comment (dustinspecker) [reply CHANGELOG.md]: Deleted and push new commit.

**Comment by dustinspecker** [comment]
Thank you for reviewing this!

**Review by jkodroff — APPROVED** [review]

**Pull request merged**